### ISSUE TITLE
pypi installation broken

### ISSUE THREAD

**Issue by Evidlo** [pschmitt/pykeepass#98]
We need to add `pykeepass.kdbx_parsing` to `packages` in setup.py.

**Comment by Evidlo** [comment]
Because of #83 .  Our test databases are so small that it doesn't matter.

It might matter eventually if we start putting icons or other binary data in the test dbs.

**Comment by pschmitt** [comment]
Right, but the KDBX files are not included, even with `find_packages()`:

```
$ tar -ztvf dist/pykeepass-3.0.1.tar.gz
drwxr-xr-x pschmitt/users    0 2018-09-07 08:23 pykeepass-3.0.1/
-rw-r--r-- pschmitt/users 11773 2018-09-07 08:23 pykeepass-3.0.1/PKG-INFO
-rw-rw-r-- pschmitt/users  9330 2018-09-06 14:09 pykeepass-3.0.1/README.rst
drwxr-xr-x pschmitt/users     0 2018-09-07 08:23 pykeepass-3.0.1/pykeepass/
-rw-r--r-- pschmitt/users    72 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/__init__.py
-rw-r--r-- pschmitt/users  5701 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/baseelement.py
-rw-r--r-- pschmitt/users  8217 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/entry.py
-rw-r--r-- pschmitt/users  3364 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/group.py
-rw-r--r-- pschmitt/users  1149 2018-05-31 07:44 pykeepass-3.0.1/pykeepass/icons.py
drwxr-xr-x pschmitt/users     0 2018-09-07 08:23 pykeepass-3.0.1/pykeepass/kdbx_parsing/
-rw-r--r-- pschmitt/users    23 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/__init__.py
-rw-rw-r-- pschmitt/users 10204 2018-09-06 14:09 pykeepass-3.0.1/pykeepass/kdbx_parsing/common.py
-rw-r--r-- pschmitt/users   781 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/kdbx.py
-rw-r--r-- pschmitt/users  4754 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/kdbx3.py
-rw-r--r-- pschmitt/users  7523 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/kdbx4.py
-rw-r--r-- pschmitt/users 15444 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/pytwofish.py
-rw-r--r-- pschmitt/users 13299 2018-08-29 10:15 pykeepass-3.0.1/pykeepass/kdbx_parsing/twofish.py
-rw-rw-r-- pschmitt/users 13702 2018-09-06 14:09 pykeepass-3.0.1/pykeepass/pykeepass.py
drwxr-xr-x pschmitt/users     0 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/
-rwxrwxr-x pschmitt/users 11773 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/PKG-INFO
-rwxrwxr-x pschmitt/users   576 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/SOURCES.txt
-rwxrwxr-x pschmitt/users     1 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/dependency_links.txt
-rwxrwxr-x pschmitt/users    56 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/requires.txt
-rwxrwxr-x pschmitt/users    16 2018-09-07 08:23 pykeepass-3.0.1/pykeepass.egg-info/top_level.txt
-rw-r--r-- pschmitt/users    38 2018-09-07 08:23 pykeepass-3.0.1/setup.cfg
-rw-r--r-- pschmitt/users   572 2018-09-07 08:23 pykeepass-3.0.1/setup.py
drwxr-xr-x pschmitt/users     0 2018-09-07 08:23 pykeepass-3.0.1/tests/
-rw-r--r-- pschmitt/users     0 2018-05-31 07:44 pykeepass-3.0.1/tests/__init__.py
-rw-rw-r-- pschmitt/users 18317 2018-09-06 14:09 pykeepass-3.0.1/tests/tests.py
```

**Comment by pschmitt** [comment]
See 81bd6d8453e95db97cbf6ae885ae36ea78fad507

**Comment by pschmitt** [comment]
Closing this now. Fixed in version 3.0.1+ (f6dcbc3)
